### DISCUSSION TITLE
Keep Relative Paths After Building

### DISCUSSION THREAD

**Discussion by simonferndriger** [parcel-bundler/parcel#9173] [Q&A]
My project is temporarily in a subfolder. So I reference all source with a relative path (e.g. href="style.css") and use a base href to address the subfolder.

Now, when building, the base href is still there, but it doesn't work anymore, because all path are not absolute (e.g. href="/index.css...").

How can I keep these relative paths such that base href continues to work after building?

**Comment by mischnic** [comment]
`--public-url ./` might work. I think the handling of public-url relative-ness is bit inconsistent right now.
